js/packages/glimmer-idc/src/components/list-item.ts

```diff
@@ -0,0 +1,72 @@
+import Component, { hbs, tracked } from '@glimmerx/component';
+import MissingAsset from './missing-asset';
+
+interface Args {
+  listItem: ListItemObject,
+}
+
+interface ListItemObject {
+  uuid: string,
+  nid: number,
+  title: string,
+  field_description: string,
+}
+
+interface JsonApiResponse {
+  data: {}[],
+  included: {
+    attributes: {
+      uri: {
+        url: string
+      }
+    }
+  }[]
+}
+
+export default class ListItem extends Component<Args> {
+  @tracked imageUrl: string = null;
+
+  constructor(owner: unknown, args: Args) {
```

Comment:
Just curious why `owner` is separated out here, but `listItem` is defined above in the Args interface

Reply:
I'm honestly not too sure. Was working from this example:
https://discuss.emberjs.com/t/glimmer-component-args-typescript/17155/2